src/main-process/atom-window.coffee

```diff
@@ -23,6 +23,7 @@ class AtomWindow
     options =
       show: false
       title: 'Atom'
+      titleBarStyle: 'hidden'
```

Comment:
Is there any chance of this ever doing something on platforms other than OS X? If so, I think it'd be nice to specifically check for the platform here, since we're also explicitly checking it in `atom-environment`, and we don't want it to accidentally get out of sync.